test-rpc-functions/src/rpc.rs

```diff
@@ -77,6 +93,31 @@ trait RpcTestFunctionsRpc {
 
 #[async_trait::async_trait]
 impl RpcTestFunctionsRpcServer for super::RpcTestFunctionsHandle {
+    async fn genesis_pool_id(&self) -> rpc::Result<String> {
+        let (genesis_pool_id, _, _, _, _, _) = regtest_genesis_values();
+        Ok(genesis_pool_id.hex_encode())
+    }
+
+    async fn genesis_private_key(&self) -> rpc::Result<String> {
+        let (_, _, genesis_stake_private_key, _, _, _) = regtest_genesis_values();
```

Comment:
Perhaps there should be asserts that double-check that the private key(s) are compatible with the values inside genesis. This whole thing is pegged on regtest_genesis_values() being consistent with genesis, which can change in the future.